samples/common/src/main/java/com/bumble/appyx/samples/common/profile/Profile.kt

```diff
@@ -0,0 +1,96 @@
+package com.bumble.appyx.samples.common.profile
+
+import android.os.Parcelable
+import androidx.annotation.DrawableRes
+import com.bumble.appyx.samples.common.R
+import kotlinx.parcelize.Parcelize
+import kotlin.random.Random
+
+@Parcelize
+@Suppress("MagicNumber")
+data class Profile(
```

Comment:
`Stable`/`Immutable`? Have you run the Compose compiler metrics on this change? worried it might introduce a non-skippable Composable.